config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -46,8 +47,9 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --cluster=e2e-big-beta
-      - --env-file=jobs/env/ci-kubernetes-e2e-scalability-common.env
-      - --env-file=jobs/env/ci-kubernetes-e2e-scalability-old-common.env
+      # Use default kube-qps and kube-api-burst flag values.
+      - --env=CONTROLLER_MANAGER_TEST_ARGS=--profiling
+      - --env=SCHEDULER_TEST_ARGS=--profiling
```

Comment:
I think it's ok to explicitly set these two, as when we advance k8s versions we can get rid of them per release-branch.